app/decorators/user_decorator.rb

```diff
@@ -7,21 +7,10 @@ def status
     "Inactive"
   end
 
-  # If all of a volunteers cases are not transition youth eligible, then
-  # we return "No", otherwise they have at least one transition youth eligible case
-  # and we return "Yes"
-  def assigned_to_transition_aged_youth?
-    volunteer_no_transition_youth_cases = object.casa_cases.pluck(:transition_aged_youth).all? false
+  def name
+    return object.email if object.display_name.blank?
 
-    volunteer_no_transition_youth_cases ? "No" : "Yes 🐛🦋"
-  end
-
-  def last_contact_made
-    if object.most_recent_contact.nil?
-      "None ❌"
-    else
-      object.most_recent_contact.occurred_at.strftime(DateFormat::FULL)
-    end
+    object.display_name
```

Comment:
This is not needed, decorator will "fall through" to allow the  `display_name` method on the underlying object to be called. It would be better to call `display_name` in the UI than alias it here.

Reply:
@compwron I think this was a mistake; I didn't end up using any decorators at `#index`.